sdk/servicebus/azure-messaging-servicebus/src/test/java/com/azure/messaging/servicebus/ServiceBusReceiverAsyncClientIntegrationTest.java

```diff
@@ -24,11 +24,13 @@
 
 class ServiceBusReceiverAsyncClientIntegrationTest extends IntegrationTestBase {
     private static final String CONTENTS = "Test-contents";
+    private static final String SESSION_ID = "seattle-id1";
```

Comment:
This wouldn't work because we need a session enabled queue. You'd have to update the ARM template to create one.